profiles/targets/clang/package.accept_keywords

```diff
@@ -0,0 +1,10 @@
+# Copyright 1999-2016 Gentoo Foundation.
+# Distributed under the terms of the GNU General Public License, v2
+# $Id$
+
+# we need the latest version of these packages
+sys-devel/llvm ~amd64
+sys-devel/clang ~amd64
+sys-libs/libcxx ~amd64
+sys-libs/libcxxrt ~amd64
+sys-libs/llvm-libunwind ~amd64
```

Comment:
Using `~*` has an unwanted effect: some experimental packages that are newer than those in the musl overly are pulled in.

For example, gcc-5.4.0 is pulled in and it overrides gcc-4.9.3-r99 in musl overlay. Consequently, gcc fails to build on musl.